From e468f06e5afd758f55f71b4ef1e31b0e48d81cd6 Mon Sep 17 00:00:00 2001
From: "Andrew F. Davis" <afd@ti.com>
Date: Tue, 4 Dec 2018 11:08:31 -0600
Subject: [PATCH 01/10] arm: K3: Increase SYSFW maximum size for HS

On HS devices, the SYSFW image certificate needs to be double
signed, this makes the image larger than the current default
maximum size. Change K3_SYSFW_IMAGE_SIZE_MAX default to allow
HS SYSFW images.

Fix typo while here.

Signed-off-by: Andrew F. Davis <afd@ti.com>
---
 arch/arm/mach-k3/Kconfig | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-k3/Kconfig b/arch/arm/mach-k3/Kconfig
index b71d265..ff31c65 100644
--- a/arch/arm/mach-k3/Kconfig
+++ b/arch/arm/mach-k3/Kconfig
@@ -120,9 +120,9 @@ config K3_SYSFW_IMAGE_MMCSD_RAW_MODE_PART
 config K3_SYSFW_IMAGE_SIZE_MAX
 	int "Amount of memory dynamically allocated for loading SYSFW blob"
 	depends on K3_LOAD_SYSFW
-	default	268000
+	default	269000
 	help
-	  Amount of memory reserved through dynamic allocation at runtimne for
+	  Amount of memory reserved through dynamic allocation at runtime for
 	  loading the combined System Firmware and configuration image tree
 	  blob. Keep it as tight as possible, as this directly affects the
 	  overall SPL memory footprint.
-- 
2.7.4

